src/components/inputForm.js

```diff
@@ -1,36 +1,41 @@
-import React from "react";
-import { useDispatch } from "react-redux";
-import { useState } from "react";
-import  { newBookFetch, postBook } from "../redux/Books/Books";
-import { v4 as  uuidv4 } from 'uuid';
+/* eslint-disable implicit-arrow-linebreak */
+import React from 'react';
+import './styles/Form.css';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { postBook } from '../redux/books/books';
 
 const Form = () => {
-    const dispatch = useDispatch();
-
-    const [title, setTitle] = useState('');
-    const [author, setAuthor] = useState('');
-   
-
-    const newBooks = (e) => {
-        e.preventDefault();
-         const bookInfo = { item_id: uuidv4(),title: title, author: author, category:'TUNJI'};
-         console.log(bookInfo);
-        dispatch(newBookFetch(bookInfo));
-
-        setTitle('')
-        setAuthor('')
-    }
-
-    <h2 className="add a new book">
-        ADD NEW BOOK
-    </h2>
-    return (
-    <form action="#" className="form">
-        <input type="text"  className="title" value={title} placeholder="title" onChange={(e) => setTitle(e.target.value)} ></input>
-        <input type="text"  className="author" value={author} placeholder="author" onChange={(e) => setAuthor(e.target.value)} ></input>
-        <button type="submit" value="submit" className="button" onClick={newBooks}>ADD BOOK</button>
-    </form>
-    )
+  const dispatch = useDispatch();
+  const handleAdd = (e) => {
+    e.preventDefault();
+    const newBook = {
+      item_id: uuidv4(),
+      title: e.target.title.value.trim(),
+      author: e.target.author.value.trim(),
+      category: e.target.category.value.trim(),
+    };
+    dispatch(postBook(newBook));
+    e.target.reset();
+  };
+  return (
+    <div className="form-container">
+      <h3>ADD NEW BOOK</h3>
+      <form className="my-form" onSubmit={handleAdd}>
+        <input type="text" name="title" placeholder="Book title" id="title" required />
+        <input type="text" name="author" placeholder="Book Author" id="author" required />
+        <select name="category" id="category" className="selectCategory">
+          <option>Select Category...</option>
+          <option value="Technology">Technology</option>
+          <option value="Fiction">Fiction</option>
+          <option value="Adventure">Adventure</option>
+          <option value="Action">Action</option>
+          <option value="Romance">Romance</option>
+        </select>
+        <button className="submit-btn" type="submit">Add Book</button>
+      </form>
+    </div>
+  );
```

Comment:
- [ ] I can't see any form on the page. Kindly fix it.

![image](https://user-images.githubusercontent.com/38635158/187011253-5058845e-af47-42e0-b221-cd4a247b106d.png)